static/js/zamboni/devhub.js

```diff
@@ -364,6 +364,10 @@ function addonFormSubmit() {
                     var e = $(format('<b class="save-badge">{0}</b>',
                                      [gettext('Changes Saved')]))
                               .appendTo(parent_div.find('h3').first());
+                    // Fix bug 7504. If the basic information has been changed. Refresh the name of the addon
+                    if ($form.is('#addon-edit-basic')) {
+                        refreshAddonBasicInformation($form);
```

Comment:
As this only updates a part of the basic information (the name), might as well just do it all here instead of having a separate function (that does less than you might think?)

Reply:
@wagnerand I though that highlighting the code that updates the basic information section by putting it in a separate function would be useful especially if one day there will be an endpoint to retrieve such updated basic information. But if this is the preferred style I am happy to stick with it :)